cluster/configure_resources.sls

```diff
@@ -1,7 +1,34 @@
 {%- from "cluster/map.jinja" import cluster with context -%}
 {% set host = grains['host'] %}
 
+include:
+  - .configure_sbd_resource
+
 {% if cluster.configure is defined %}
+
+# Configure the cluster properties
+{% if cluster.configure.properties is defined and cluster.configure.properties|length>0 %}
+configure_cluster_properties:
+  crm.cluter_properties_present:
+    - properties: {{ cluster.configure.properties|json }}
```

Comment:
Are we passing it `json` type here for properties, if so, we are again converting in salt-shaptools functions to json data
https://github.com/SUSE/salt-shaptools/blob/804c8df8e5f7b68420aacbe44b7b2da285b50271/salt/modules/crmshmod.py#L636

Reply:
This json is useful to standardize booleans and strings. We could actually remove it most probably, but I think it's not harmful, and make visible that this entry should be for json format